test/verify/check-journal

```diff
@@ -295,7 +295,7 @@ s.send("PRIORITY=3\\nFOO=bar\\n")'
 
         b.wait_text("#journal-entry-message", "[no data]")
 
-    @skipImage("Newer version of ABRT required", "centos-7", "rhel-7", "fedora-25", "fedora-i386", "fedora-testing", "rhel-7-4")
+    @skipImage("Newer version of ABRT required", "centos-7", "rhel-7", "fedora-25", "fedora-i386", "fedora-testing", "rhel-7-4", "rhel-7-5")
```

Comment:
Did you try running this? Shouldn't a working version have gotten here by now?

Reply:
I did try this, and added the skip after seeing that it's not working with the current version in 7.5 yet.

Reply:
Ok, i guess we should ping them about it and find out what their plan is.